modules/kafka-cluster/inputs.tf

```diff
@@ -30,5 +23,20 @@ variable "region" {
   description = "The cloud service provider region where the Kafka cluster is running - see supported list - https://docs.confluent.io/cloud/current/clusters/regions.html#cloud-providers-and-regions"
   type        = string
   default     = "asia-southeast2"
+}
 
+variable "cluster_type" {
+  description = "The value can be either basic, standard, and dedicated."
+  type        = string
+  default     = "basic"
+}
+variable "cku" {
```

Comment:
Minor: This is relevant only for dedicated, right? So maybe we could name it `dedicated_cluster_cku` ? Just thinking. Or in the first sentence of the description say "Relevant only for dedicated cluster type"

Reply:
this is also done